modules/openapi-generator/src/main/resources/python-aiohttp/requirements.mustache

```diff
@@ -1,3 +1,3 @@
-connexion[aiohttp,swagger-ui] == 2.0.2
-swagger-ui-bundle == 0.0.2
-aiohttp_jinja2 == 1.1.0
+connexion[aiohttp,swagger-ui] == 2.6.0
```

Comment:
This generator states that is supports  python 3.5 but this version of connexion does not work in python 3.5
Can you add the connexion support for python >= 3.5 here?